.pre-commit-config.yaml

```diff
@@ -2,35 +2,23 @@
 # See https://pre-commit.com/hooks.html for more hooks
 default_language_version:
     python: "python3.12"
-ci:
-    autoupdate_schedule: quarterly
```

Comment:
These two lines are configuration for pre-commit's managed CI service, pre-commit.ci, only. I dropped them because you don't currently use the service.